_docs_integrate/version-5-migration-guide.md

```diff
@@ -21,85 +21,98 @@ required_by:
+## Step-by-Step Instructions
 
-## Removal of RelationshipChanges
+The step-by-step instructions can be consulted to start the migration to version 5 directly.
+More [detailed explanations]({% link _docs_integrate/version-5-migration-guide.md %}#detailed-explanations) can be found below.
 
-RelationshipChanges were removed. The only type of RelationshipChange used before was the CreationChange, its `content` has been moved to a new property of the relationship - the `creationContent`. If this content has been created by responding to a [Request]({% link _docs_integrate/data-model-overview.md %}#request) of a [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate), the [content]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplatecontent) changes as follows: The `<relationshipChange>.request.content` value was of type `RelationshipCreationChangeRequestContent`, the `<relationship>.creationContent` is now of type `RelationshipCreationContent`.
+### Connector Setup
 
-The Connector route for creating the [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) is unchanged, the Connector routes using RelationshipChanges (accept, reject RelationshipChange) were replaced with corresponding routes on the Relationship. No `content` must be sent with any of those operations. The operations are recorded in the new `auditLog` property of the Relationship.
+- The data from the database that was used by the Connector of the former version is outdated. This is because the [format of addresses has changed to DIDs]({% link _docs_integrate/version-5-migration-guide.md %}#dids-as-addresses), for example. For this reason, the old data must be deleted. Alternatively, the database can be deleted as a whole and [set up again]({% link _docs_operate/setup-with-docker-compose.md %}).
+- The [image](https://github.com/nmshd/connector?tab=readme-ov-file#connector) used to build the Connector must be updated to version 5.
+- Some changes must be made to the [configuration]({% link _docs_operate/configuration.md %}) of the Connector.
+  - It must be ensured that a [Backbone](https://github.com/nmshd/backbone/tags) is used which is compatible with version 5 of the Connector. This means that a Backbone of version 6 must be used. Therefore, specify appropriate [Backbone credentials]({% link _docs_operate/configuration.md %}#transportlibrary) in the configuration.
+  - The AutoAcceptRelationshipCreationChangesModule must be renamed to the [AutoAcceptPendingRelationshipsModule]({% link _docs_operate/configuration.md %}#autoacceptpendingrelationships), because the [RelationshipChanges have been removed]({% link _docs_integrate/version-5-migration-guide.md %}#removal-of-relationshipchanges). In addition, the `responseContent` property of the Module must be removed in its configuration.
+  - The WebhooksV2Module must be renamed to the [WebhooksModule]({% link _docs_operate/configuration.md %}#webhooks).
```

Comment:
This information is not entirely correct. We have renamed the module some time ago but had thingy in place to make this change backwards compatible. THIS was removed :)

Also - please say what **exactly** this means (renaming the field `modules.webhooksV2` to `modules.webhooks` IF you have specified and not already migrated to `webhooks`

Reply:
Good comment, I had also noticed in the changes of the code that there was an error message before if you used `webhooksV2` instead of `webhooks` and that this has now been removed.

I will incorporate it!